app/(saas)/projects/[projectSlug]/hooks/use-get-od-news.tsx

```
@@ -0,0 +1,32 @@
+  });
+
+  const response = await fetch(url.toString());
+
+  const data = await response.json();
+
+  return data?.data?.map((n: Submission) => new OdNewsModel(n)) ?? [];
+};
+
+export function useGetOdNews({ projectId, limit = "10" }: { projectId?: string; limit?: string }) {
+  return useQuery({
+    queryKey: ["od-news", projectId],
```

ESLint

[error] The following dependencies are missing in your queryKey: limit

(@tanstack/query/exhaustive-deps)